DeAceroDLabs/marketplace-frontend
# Feature/dmp 602 Generate Devops Subtree

## What is done

[[DMP-602] Generate Devops Subtree](https://deacero.atlassian.net/browse/DMP-602)

https://github.com/DeAceroDLabs/DevOps/tree/master/docker
Followed the DevOps standar.


## How is done

- Followed the steps indicated on the DevOps repo, updated set_vars.sh values.
- Github actions will be added on another PR.
- Obtained permission to see Amazon Elastic Containers Registry Repositories:

![image](https://user-images.githubusercontent.com/81311646/194432377-78fa7bd6-f467-4874-bac2-c10db669a89d.png)

- Created repository for back with the same name as DOCKER_PROJECT_NAME on set_vars.sh:
![image](https://user-images.githubusercontent.com/81311646/194432728-b8decb05-15d0-4b4f-b4a2-1104aa9cd438.png)


## How to test

1. Do first the steps on back PR: https://github.com/DeAceroDLabs/mkt-place-back/pull/16
2. Run `npm run docker:run`
3. Go to `localhost:8300`.
4. You should see everyhting working.
